### DISCUSSION TITLE
o.e.c.Authenticated - Authentication failure

### DISCUSSION THREAD

**Discussion by marianoka** [Security-Onion-Solutions/securityonion#5018] [General]
hi !!
I'm getting a Request Failed 400 code while trying to scale from alerts to the hive. Reading logs, I have seen this lines
[error] o.e.c.Authenticated - Authentication failure:
        session: AuthenticationError User session not found
        pki: AuthenticationError Certificate authentication is not configured
        key: AuthenticationError Authentication failure
        init: AuthenticationError Use of initial user is forbidden because users exist in database
this issue starts after and update.
Any idea??

**Comment by weslambert** [comment]
Are you still experiencing this issue?
